src/content/docs/nrql/nrql-tutorials/nrql-tutorial-advanced-dashboarding.mdx

```diff
@@ -220,9 +221,9 @@ These queries further advance your NRQL ability. Apdex is an industry standard a
 
 ### Use extrapolate for large amounts of data
 
-The New Relic Database (NRDB) receives and processes large amounts of data every day at lightning speed! When APM records a large amount of event data, New Relic agents implement a sampling technique to continue collecting meaningful data while reducing potential impact to your applications. This usually only happens when a single event in an application or service handles extremely high volumes of requests. If you have multiple agents spread across multiple load-balanced instances of a service, you may never even observe this limit.
+The New Relic Database (NRDB) receives and processes large amounts of data every day at lightning speed! When APM records a large amount of event data, New Relic agents implement a sampling technique to continue collecting meaningful data while reducing potential impact to your applications. This usually only happens when a single event in an application or service handles extremely high volumes of requests. 
 
-Let's discuss what you can do when this happens. The `EXTRAPOLATE` operator tells New Relic to mathematically compensate for the effects of sampling, thereby returning results that more closely represent activity in your system. We store an extra value to record how many similar events occured over the limit, which allows us to deliver statistically accurate results. 
+If you have multiple agents spread across multiple load-balanced instances of a service, you may never even observe this limit. Let's find out what you can do when this happens. The `EXTRAPOLATE` operator tells New Relic to mathematically compensate for the effects of sampling, thereby returning results that more closely represent activity in your system. We store an extra value to record how many similar events occured over the limit, which allows us to deliver statistically accurate results. 
```

Comment:
We and us is sort of strange because the NR part of the we/us isn't doing anything. It's up to the reader to take action or not

Reply:
Agreed and changed